test/aws/osml/model_runner/tile_worker/test_variable_tile_tiling_strategy.py

```diff
@@ -6,17 +6,27 @@
 
 class TestVariableTileTilingStrategy(TestCase):
     def test_compute_regions(self):
+        """
+        Test that regions are correctly computed for a full-sized image based on
+        the specified nominal region size, tile size, and overlap.
+        """
         from aws.osml.model_runner.tile_worker import VariableTileTilingStrategy
 
         tiling_strategy = VariableTileTilingStrategy()
 
+        # Define image and tiling parameters
         full_image_size = (25000, 12000)
         nominal_region_size = (10000, 10000)
         overlap = (100, 100)
         tile_size = (4096, 4096)
 
+        # Compute regions
         regions = tiling_strategy.compute_regions(((0, 0), full_image_size), nominal_region_size, tile_size, overlap)
+
+        # Verify the number of computed regions
         assert len(regions) == 6
+
+        # Verify that all computed regions match expected results
```

Comment:
again, not sure these are particularly insightful comments to add.  Adding the docstring is good to describe the tests but was the code here confusing without comments?